.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
```

Comment:
Would you like to get reports about the issue from our bugs?
Or, get reports of any problems you need help with?

Reply:
I would like the latter. I will change to 'Issue Report' for these templates. Thank you